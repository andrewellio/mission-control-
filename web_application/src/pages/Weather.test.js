const weather = require("./Weather");

test("Testing Jest", () => {
  expect(true).toBeTruthy();
});
