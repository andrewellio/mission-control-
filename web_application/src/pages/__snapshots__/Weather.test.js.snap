// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Weather />  with no props should match the snapshot 1`] = `"<a class=\\"Instructions-back\\" href=\\"/\\"><i class=\\"material-icons\\">navigate_before</i></a><div class=\\"Landing-body\\"><h1>Weather</h1><p id=\\"weatherdata\\">Please enter the required fields below</p><label class=\\"space\\"> </label><br/><label class=\\"latValue\\">Latitude:</label><br/><input type=\\"int\\" placeholder=\\"e.g. -41.2769\\" id=\\"latId\\" value=\\"0\\"/><br/><label class=\\"space\\"> </label><br/><label class=\\"longValue\\">Longitude:</label><br/><input type=\\"int\\" placeholder=\\"e.g. 174.7731\\" id=\\"lonId\\" value=\\"0\\"/>Â <button class=\\"button\\" id=\\"getData\\">Get Data</button><button class=\\"button\\" id=\\"saveData\\">Save Data</button><button class=\\"button\\" id=\\"loadData\\">Load Data</button><input type=\\"file\\" accept=\\".json\\"/></div>"`;
